### PR TITLE
Fix/moves env vars

### PR DESCRIPTION
**Pull request by t0ff3r**
## What is the reason for this change?

To be able to share environment variables more easily between packages

### What is the expected result?

Easily share environment variables

## How is it solved?

Moving environment variables to the root of our project

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |  x       |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] This needs a thorough regression test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,37 +30,25 @@ Install all `packages`
...
+ELVID_REDIRECT_URI=http://localhost:8000/auth/signin
+ELVID_SCOPE=openid profile email kunde.kundeportalapi
+CONTENTFUL_ACCESS_TOKEN=
+CONTENTFUL_PREVIEW_TOKEN=***
+CONTENTFUL_MANAGEMENT_TOKEN=***
+CONTENTFUL_SPACE_ID=***
+CONTENTFUL_ENVIRONMENT=master
+KUNDEPORTAL_API_URL=***
+POWEROUTAGE_API_URL=***
+```
```
Comment (FredricW) [inline README.md:51]:
```suggestion
GOOGLE_MAPS_TOKEN=***
```
```

File: README.md [hunk 2]
```diff
@@ -30,37 +30,25 @@ Install all `packages`
...
+ELVID_CLIENT_ID=***
+ELVID_REDIRECT_URI=http://localhost:8000/auth/signin
+ELVID_SCOPE=openid profile email kunde.kundeportalapi
+CONTENTFUL_ACCESS_TOKEN=
+CONTENTFUL_PREVIEW_TOKEN=***
+CONTENTFUL_MANAGEMENT_TOKEN=***
+CONTENTFUL_SPACE_ID=***
+CONTENTFUL_ENVIRONMENT=master
+KUNDEPORTAL_API_URL=***
+POWEROUTAGE_API_URL=***
```
Comment (FredricW) [inline README.md:50]:
```suggestion
POWEROUTAGE_API_URL=***
GOOGLE_MAPS_TOKEN=***
```
Denne manglet i forrige versjon av doksa, men er nødvendig for skjema komponenten og alt annet som må bruke Google Maps komponenten.

File: README.md [hunk 3]
```diff
@@ -106,11 +93,6 @@ yarn web:test
 
 # Hdd-React
```
Comment (FredricW) [inline README.md:95]: Leave no explanation of what it is?

**Review by FredricW — CHANGES_REQUESTED** [review]
Looks good to me! Only some comments about the readme :)

**Review by JonatanPaalsson — APPROVED** [review]
Have done some testing and can't find any problems there 👍

**Review by FredricW — APPROVED** [review]

**Pull request merged**